src/Components/App/App.js

```diff
@@ -98,31 +96,18 @@ class App extends Component {
   }
 
   handleCharacterHomeworld = (character) => {
-    return fetch(character.homeworld)
```

Comment:
This definitely cleans up these functions and this component a lot - makes it a lot more readable